akka-client/src/main/scala/com/qvantel/jsonapi/client/akka/AkkaClient.scala

```diff
@@ -58,14 +57,14 @@ object AkkaClient {
     new JsonApiClient[A] {
       implicit val config: UriConfig = uriConfig
 
-      override def one(id: String, include: Set[String] = Set.empty): IO[Option[A]] =
+      override def one(id: String, include: Set[String] = Set.empty)(implicit pt: PathToId[A]): IO[Option[A]] =
```

Comment:
Why is `PathToId` a parameter to this method, since you already know `A` at `instance`?

Reply:
Because that is there in the `JsonApiClient` this extends from.

Reply:
Are the constraints really needed at method level, though? Either the methods and the trait could be unconstrained, with concrete implementations then requiring them or `JsonApiClient` could be defined as:

```scala
abstract class JsonApiClient[A](implicit pt: PathToId[A], writer: JsonApiWriter[A])
```

Or do you expect to need to be able to provide different instances of `PathToId` and `JsonApiWriter` per method call?

Reply:
Main difference is that some of them only require a normal `PathTo` (post for example) and some `PathToId`

Reply:
Decided to remove the type param form JsonApiClient and move all the implicits to the methods.